templates/_base.html

```diff
@@ -23,6 +23,9 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
 
   <meta name="theme-color" content="#366597">
+
+  <link rel="stylesheet" href="/static/css/base.css" />
```

Comment:
Since you have this, can you remove line 45? (the one that calls inline_file on main.css)

Reply:
But base.css doesn't include main.css.  We could probably also append main.css in base.css.  How about I do that in another PR?

Reply:
Ah, my mistake.  I thought that your new "css" rule was concatenating two things together, but it is really just copying one source to one destination.